Milter/utils.py

```diff
@@ -135,9 +135,9 @@ def parseaddr(t):
   >>> parseaddr('God@heaven <@hop1.org,@hop2.net:jeff@spec.org>')
   ('God@heaven', 'jeff@spec.org')
   >>> parseaddr('Real Name ((comment)) <addr...@example.com>')
-  ('Real Name', 'addr...@example.com')
+  ('Real Name (comment)', 'addr...@example.com')
```

Comment:
Ok, your right.  This, "Real Name (comment)" is pretty innocuous, and maybe even correct.